packages/core/src/amazonqFeatureDev/controllers/chat/controller.ts

```diff
@@ -202,6 +209,21 @@ export class FeatureDevController {
         }
     }
 
+    private async processChatItemFeedbackMessage(message: any) {
+        const session = await this.sessionStorage.getSession(message.tabId)
+
+        await globals.telemetry.postFeedback({
+            comment: `${JSON.stringify({
+                type: 'featuredev-chat-answer-feedback',
+                conversationId: session?.conversationId ?? '',
+                messageId: message?.messageId,
+                reason: message?.selectedOption,
+                userComment: message?.comment,
+            })}`,
+            sentiment: 'Negative',
```

Comment:
Can it only ever be negative?

Reply:
Yes, Mynah UI only allows users report feedback when they click on `thumb down ` in the message, so it is always Negative.

Reply:
Ok. Let's add a comment:

```suggestion
            sentiment: 'Negative', // The chat UI reports only negative feedback currently.
```